includes/wf_crm_webform_postprocess.inc

```diff
@@ -2123,8 +2123,11 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
 
       $lineItemRecord = json_decode(json_encode($item), FALSE);
       // Add financial_item and entity_financial_trxn
-      $taxTrxnID = !empty($lineItemRecord->tax_amount);
-      CRM_Financial_BAO_FinancialItem::add($lineItemRecord, $contributionResult, $taxTrxnID);
+      CRM_Financial_BAO_FinancialItem::add($lineItemRecord, $contributionResult);
```

Comment:
Can you add a comment here to explain why we need to call `CRM_Financial_BAO_FinancialItem::add` twice (I understand now, but I didn't at the time as the code is not really clear on the CiviCRM side).